src/new/wallet.js

```diff
@@ -765,44 +765,128 @@ class HathorWallet extends EventEmitter {
    * Send a transaction from its outputs
    * Currently does not have support to send custom tokens, only HTR
    *
-   * @param {Array} outputs Array of outputs with each element as an object with {'address', 'value'}
-   * @param {Array} inputs Array of inputs with each element as an object with {'hash', 'index'}
+   * @param {Array} outputs Array of outputs with each element as an object with {'address', 'value', 'timelock', 'token'}
+   * @param {Array} inputs Array of inputs with each element as an object with {'hash', 'index', 'token'}
    * @param {Object} token Token object {'uid', 'name', 'symbol'}. Optional parameter if user already set on the class
    * @param {Object} options Options parameters
    *  {
    *   'changeAddress': address of the change output
+   *   'startMiningTx': boolean to trigger start mining (default true)
    *  }
    *
    * @return {Promise} Promise that resolves when transaction is sent
    **/
-  sendManyOutputsTransaction(outputs, inputs = [], token = null, options = { changeAddress: null }) {
-    // XXX To accept here multi tokens in the same tx would be a bit more complicated because
-    // the method prepareSendTokensData would need a bigger refactor.
-    // I believe we should refactor all of that code (and it will be done on wallet-service)
-    // so I decided to change this method to accept only one token, which is enough for our needs right now
+  sendManyOutputsTransaction(outputs, inputs = [], token = null, options = {}) {
```

Comment:
Maybe rename to `defaultToken`?

Reply:
This will change in the branch (already started) to make the facades method signatures uniform. I don't think we should change this here also.

What do you think?